bitcoin/src/amount.rs

```diff
@@ -83,8 +83,28 @@ impl Denomination {
             Denomination::MilliSatoshi => "msat",
         }
     }
+
+    /// The different str forms of denominations that are recognized.
+    fn forms(s: &str) -> Option<Self> {
+        match s {
+            "BTC" | "btc" | "BTCs" | "btcs" => Some(Denomination::Bitcoin),
+            "cBTC" | "cbtc" | "cBTCs" | "cbtcs" => Some(Denomination::CentiBitcoin),
+            "mBTC" | "mbtc" | "mBTCs" | "mbtcs" => Some(Denomination::MilliBitcoin),
+            "uBTC" | "ubtc" | "uBTCs" | "ubtcs" => Some(Denomination::MicroBitcoin),
+            "nBTC" | "nbtc" | "nBTCs" | "nbtcs" => Some(Denomination::NanoBitcoin),
+            "pBTC" | "pbtc" | "pBTCs" | "pbtcs" => Some(Denomination::PicoBitcoin),
```

Comment:
Why do we allow plural for anything other than sats and bits?

Reply:
I would prefer no plural for these ones.